game-core/src/main/java/games/strategy/engine/chat/ChatPlayerPanel.java

```diff
@@ -238,24 +239,11 @@ private String getDisplayString(final ChatParticipant chatParticipant) {
       return "";
     }
 
-    String extra = chatParticipant.isModerator() ? " " + TAG_MODERATOR : "";
-    String status = chat.getStatus(chatParticipant.getPlayerName());
+    final String extra = chatParticipant.isModerator() ? " " + TAG_MODERATOR : "";
+    final String status = Ascii.truncate(chat.getStatus(chatParticipant.getPlayerName()), 25, "");
```

Comment:
Have you had any thoughts whether chat truncation be done at the client, just the server, or both server and client?

Reply:
I believe @ssoloff dealt with sanitation before, maybe he knows something about it

Reply:
Sorry, got a bit rhetorical. The truncation probably should be done in chat controller, unfortunately chat messages are sent via broad cast, so they are blindly forwarded.